### ISSUE TITLE
logging configuration

### ISSUE THREAD

**Issue by wyli** [Project-MONAI/model-zoo#322]
> Currently, bundles usually leverage this line config to run with correct logging:
> https://github.com/Project-MONAI/model-zoo/blob/dev/models/spleen_ct_segmentation/configs/logging.conf#L11
> Please show the recommendation change or how to update the existing bundles?
> CC @yiheng-wang-nv .
> 
(from https://github.com/Project-MONAI/MONAI/pull/6051#issuecomment-1445631476)

with the `StatsHandler` update, `configs/logging.conf` (which is mainly used to add a stdout handler to the root logger) can be removed. But all the `StatsHandler` will need a `name=a_logger_name` to log to stdout. `configs/logging.conf` is still useful for the other log handling use cases such as  adding a `logging.FileHandler`

**Comment by Nic-Ma** [comment]
Hi @wyli ,

I think `name=a_logger_name` is only necessary if we removed `configs/logging.conf`, right?
So it's a non-breaking change.
We still need the global logging level to enable the logging print from ignite engine itself unless we set the logging of engine manually when expressions.

Thanks.

**Comment by wyli** [comment]
yes, I think we can keep `configs/logging.conf`, changing the level to `DEBUG` can show some debug info from the engine.

**Comment by yiheng-wang-nv** [comment]
Hi @wyli , sorry for late reply, do you mean modify:
```
[logger_root]
level=INFO
handlers=consoleHandler
```
into
```
[logger_root]
level=DEBUG
handlers=consoleHandler
```
for each bundle?
